zebra-network/Cargo.toml

```
@@ -24,11 +24,12 @@ serde = { version = "1", features = ["serde_derive"] }
 thiserror = "1"
 
 futures = "0.3"
-tokio = { version = "0.3.6", features = ["net", "time", "stream", "tracing", "macros", "rt-multi-thread"] }
-tokio-util = { version = "0.5", features = ["codec"] }
-tower = { version = "0.4", features = ["retry", "discover", "load", "load-shed", "timeout", "util", "buffer"] }
+tokio = { version = "1.12.0", features = ["net", "time", "tracing", "macros", "rt-multi-thread"] }
+tokio-stream = { version = "0.1.7", features = ["sync", "time"] }
+tokio-util = { version = "0.6", features = ["codec"] }
+tower = { version = "0.4.9", features = ["retry", "discover", "load", "load-shed", "timeout", "util", "buffer"] }
 
-metrics = "0.13.0-alpha.8"
+metrics = "0.17"
```

Let's use full versions, to avoid duplicate dependencies.

```suggestion
metrics = "0.17.0"
```

---

Done.